sqltelemetry: append Counter to end of Counter names

otan opened the pull request:
Doing some spring cleaning of consistency in the sqltelemetry package by
making sure all counters end with the word `Counter`.

Release justification: telemetry / small refactor with no user facing changes.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/46013)
<!-- Reviewable:end -->

rohany commented:
LGTM

probably going to have some conflicts with some other telemetry PRs merging in

otan committed:
sqltelemetry: append Counter to end of Counter names

Doing some spring cleaning of consistency in the sqltelemetry package by
making sure all counters end with the word Counter.

Release justification: telemetry / small refactor with no user facing changes.

Release note: None

otan commented:
bors r=rohany
